src/main/drivers/pwm_output.h

```diff
@@ -30,11 +30,27 @@ typedef enum {
     PWM_TYPE_DSHOT600,
     PWM_TYPE_DSHOT300,
     PWM_TYPE_DSHOT150,
+    PWM_TYPE_DSHOT900,
+    PWM_TYPE_DSHOT1200,
     PWM_TYPE_MAX
```

Comment:
To me the existing list with DShot600 first always looked out of order.

We can simple remap them for older versions in the configurator right when we receive / send them, and all will be fine